<launch>
  <node pkg="test_encoders" type="main_node.py" name="nodoPrincipal" />
  <node pkg="test_encoders" type="Control2.py" name="nodo_control"/>
  <node pkg="test_encoders" type="odometry_node.py" name="nodo_odometria"/>
  <node pkg="test_encoders" type="low_level_controller.py" name="nodo_control_encoder"/>

</launch>
